src/libraries/Common/src/Internal/Cryptography/UniversalCryptoDecryptor.cs

```diff
@@ -24,7 +24,7 @@ public UniversalCryptoDecryptor(PaddingMode paddingMode, BasicSymmetricCipher ba
         {
         }
 
-        protected sealed override int UncheckedTransformBlock(byte[] inputBuffer, int inputOffset, int inputCount, byte[] outputBuffer, int outputOffset)
+        protected override int UncheckedTransformBlock(ReadOnlySpan<byte> inputBuffer, Span<byte> outputBuffer)
```

Comment:
Should/can this one be sealed, like its predecessor?

Reply:
The whole class is sealed, wouldn't that make it redundant when specified on the method?